annotations_models/train_random_forest_classifier_model.ipynb

```diff
@@ -918,28 +696,35 @@
        "  </thead>\n",
        "  <tbody>\n",
        "    <tr>\n",
-       "      <th>Sand</th>\n",
-       "      <td>0.990585</td>\n",
-       "      <td>0.945913</td>\n",
-       "      <td>0.967059</td>\n",
+       "      <th>Ground</th>\n",
+       "      <td>0.591089</td>\n",
+       "      <td>0.811671</td>\n",
+       "      <td>0.660093</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>Water</th>\n",
+       "      <td>0.921515</td>\n",
+       "      <td>0.660503</td>\n",
+       "      <td>0.732459</td>\n",
        "    </tr>\n",
        "    <tr>\n",
-       "      <th>Vegetation</th>\n",
-       "      <td>0.992399</td>\n",
-       "      <td>0.998599</td>\n",
-       "      <td>0.995464</td>\n",
+       "      <th>Waterplants</th>\n",
+       "      <td>0.703950</td>\n",
+       "      <td>0.402544</td>\n",
+       "      <td>0.425975</td>\n",
        "    </tr>\n",
        "  </tbody>\n",
        "</table>\n",
        "</div>"
       ],
       "text/plain": [
-       "            precision    recall  f1-score\n",
-       "Sand         0.990585  0.945913  0.967059\n",
-       "Vegetation   0.992399  0.998599  0.995464"
+       "             precision    recall  f1-score\n",
+       "Ground        0.591089  0.811671  0.660093\n",
+       "Water         0.921515  0.660503  0.732459\n",
+       "Waterplants   0.703950  0.402544  0.425975"
```

Comment:
This isn't as great a score as you made me believe in the call with our stakeholders ;)

Reply:
I think i said that the model works best if there has been annotations on the satellite image.
Since for PNEO we only have 3 dates, it's very unstable if we only learn on 2 dates and predict 1 date

Reply:
Ah, okay, so that's the difference. And fair enough 2 training images and 1 test image is a bit skewed in any case.